From bc896e88bf290b97773222a0ae85dcc7ddcc4b39 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 27 Aug 2021 19:01:10 +0000
Subject: [PATCH] pan/mid: enable support for Mali T628 in XU4

Enable the T620 chip ID for the T628 in the Odroid XU4.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 src/gallium/drivers/panfrost/pan_screen.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/drivers/panfrost/pan_screen.c b/src/gallium/drivers/panfrost/pan_screen.c
index 46cd97bbf13..e4aa7447ab0 100644
--- a/src/gallium/drivers/panfrost/pan_screen.c
+++ b/src/gallium/drivers/panfrost/pan_screen.c
@@ -855,6 +855,7 @@ panfrost_create_screen(int fd, struct renderonly *ro)
         /* Check if we're loading against a supported GPU model. */
 
         switch (dev->gpu_id) {
+        case 0x620: /* T620 */
         case 0x720: /* T720 */
         case 0x750: /* T760 */
         case 0x820: /* T820 */
-- 
2.17.1

